packages/webapp/webapp_server.js

```diff
@@ -886,6 +887,9 @@ function runWebAppServer() {
     const unixSocketPath = process.env.UNIX_SOCKET_PATH;
 
     if (unixSocketPath) {
+      if (cluster.isWorker) {
+        unixSocketPath += "." + cluster.worker.id.toString() + ".sock";
```

Comment:
You don't need the explicit `.toString()` call if you're concatenating with string literals.

Reply:
Fixed.